docs/limitations.md

```diff
@@ -8,10 +8,13 @@ enterprise: 'yes'
 *   DC/OS Spark only supports submitting jars and Python scripts. It
 does not support R.
 
+*   Mesosphere does not provide support for Spark app development.
+
 *   Spark jobs run in Docker containers. The first time you run a
 Spark job on a node, it might take longer than you expect because of
 the `docker pull`.
 
-*   For interactive analytics, we
+*   DC/OS Spark only supports running the Spark Shell from within the
```

Comment:
suggest "DC/OS Spark only supports running the Spark shell from within a DC/OS cluster. See the Spark Shell section for more information."